src/components/Dropdown/Dropdown.vue

```diff
@@ -1,10 +1,11 @@
 <template>
-  <button class="relative text-left focus:outline-none" @focusout="closeDropdown">
+  <button class="relative text-left focus:outline-none" :class="parentWidth || 'w-auto'" @focusout="closeDropdown">
```

Comment:
Any reason you made these changes to the dropdown? You've destroyed the mobile view.

Reply:
lol destroyed? the reason I made it is cause it wouldn't fill the parent div, and hence some text were being cut out. How did it "destroy" the mobile view?

Reply:
Huh?

I still need to fix an issue I just noticed with the dropdowns though but not with the way you changed it 😏
**Before vs after**

Before
<img width="319" alt="Screenshot 2019-12-16 at 7 00 44 PM" src="https://user-images.githubusercontent.com/20643330/70932137-4a4ce700-2039-11ea-8cf4-a1ea4a4a0aac.png">
<img width="318" alt="Screenshot 2019-12-16 at 7 17 10 PM" src="https://user-images.githubusercontent.com/20643330/70932138-4a4ce700-2039-11ea-9706-51a3b43f80c7.png">

After
<img width="328" alt="Screenshot 2019-12-16 at 6 59 39 PM" src="https://user-images.githubusercontent.com/20643330/70932156-533db880-2039-11ea-8e1c-ec27a5771786.png">
<img width="324" alt="Screenshot 2019-12-16 at 6 59 49 PM" src="https://user-images.githubusercontent.com/20643330/70932157-533db880-2039-11ea-8662-5115201966e9.png">

Reply:
Yeah, the reason why this couldn't have worked, is cause it ended up being a top page dropdown for **EVERY** dropdown used on a small screen 😓 @chinezenwosu